ci-operator/step-registry/ipi/conf/vsphere/ipi-conf-vsphere-chain.yaml

```diff
@@ -0,0 +1,9 @@
+chain:
+  as: ipi-conf-vsphere
+  steps:
+  - ref: ipi-conf-vsphere-vips
+  - ref: ipi-conf-vsphere-dns
+  - ref: ipi-conf
```

Comment:
@wking this should be the shared conf chain, right?

Reply:
I there a shared conf chain?  I think just [the ref][1] being used here.

[1]: https://github.com/openshift/release/blob/01d6cf7dfa177dcde3d20d0f6796f588adc7b958/ci-operator/step-registry/ipi/conf/ipi-conf-ref.yaml